### ISSUE TITLE
Suggestion: Mob Drops Statistics

### ISSUE THREAD

**Issue by dimitrisal** [SharpWoW/KillTrack#7]
Is it possible/doable to add mob drop statistics? For example have an average of money drop , show up to 5-10 or so most dropped items with statistics 

Example
Killed 1000 times
Money 5 S average
Silk Cloths 500 (25% drop Chance)
Green Bow what ever 1 (0.001 %)

etc etc

**Comment by Sharparam** [comment]
Should be doable given there's a reliable way to connect gold/item drops with the mob.

My only concern would be that people have a bunch of kills tracked without this extra information, which could skew stuff like "money per kill on average". Either it could show those stats with a disclaimer that they might be inaccurate for some users, or give an option to reset all stats if users want more accuracy.

That, or there might be some fancy "rolling average" algorithm that can take into account only recorded averages. Such a method would work for gold drops but be more difficult for item drops, perhaps.

**Comment by dimitrisal** [comment]
I don't think you can calculate average without resetting data.... So you can just give the option of resetting or calculating anything above the update date?

**Comment by goako** [comment]
I would also really like this feature added. In fact this is what I was lookig for when I found this addon.
